### PR TITLE
Field Report Edit/Create Page

### PR DESCRIPTION
**Pull request by tnagorra**
## This PR doesn't introduce:
- [ ] typos
- [ ] conflict markers
- [ ] unwanted comments
- [ ] temporary files, auto-generated files or secret keys
- [ ] `console.log` meant for debugging
- [ ] codegen errors

### CODE REVIEW

File: src/components/RadioInput/Radio/styles.module.css [hunk 1]
```diff
@@ -15,7 +15,7 @@
         flex-direction: column;
 
         .description {
-            color: var(--go-ui-color-white);
+            color: var(--go-ui-color-gray-70);
```
Comment (frozenhelium) [inline src/components/RadioInput/Radio/styles.module.css]:
```suggestion
            color: var(--go-ui-color-text-light);
```

File: src/views/FieldReportForm/ContextFields/styles.module.css [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.context-fields {
+}
```
Comment (frozenhelium) [inline src/views/FieldReportForm/ContextFields/styles.module.css]: Let's not leave empty stylings

File: src/views/FieldReportForm/ResponseFields/styles.module.css [hunk 3]
```diff
@@ -0,0 +1,2 @@
+.contacts-section {
+}
```
Comment (frozenhelium) [inline src/views/FieldReportForm/ResponseFields/styles.module.css]: 1. Let's not leave it empty
2. 
```suggestion
.response-fields {
}
```

File: src/views/FieldReportForm/RiskAnalysisFields/styles.module.css [hunk 4]
```diff
@@ -0,0 +1,2 @@
+.risk-analysis-fields {
+}
```
Comment (frozenhelium) [inline src/views/FieldReportForm/RiskAnalysisFields/styles.module.css]: Let not keep empty styles

**Review by frozenhelium — CHANGES_REQUESTED** [review]

File: src/views/FieldReportForm/RiskAnalysisFields/styles.module.css (hunk 4 above)
Comment (tnagorra) [reply src/views/FieldReportForm/RiskAnalysisFields/styles.module.css]: Styling is a WIP

File: src/views/FieldReportForm/ResponseFields/styles.module.css (hunk 3 above)
Comment (tnagorra) [reply src/views/FieldReportForm/ResponseFields/styles.module.css]: Styling is a WIP

File: src/views/FieldReportForm/ContextFields/styles.module.css (hunk 2 above)
Comment (tnagorra) [reply src/views/FieldReportForm/ContextFields/styles.module.css]: Styling is a WIP

File: src/components/RadioInput/Radio/styles.module.css (hunk 1 above)
Comment (tnagorra) [reply src/components/RadioInput/Radio/styles.module.css]: Yes sir!

**Review by frozenhelium — APPROVED** [review]

**Pull request merged**